### DISCUSSION TITLE
Blazor sdk - sitecore jss

### DISCUSSION THREAD

**Discussion by GoranHalvarsson** [Sitecore/jss#1408] [Ideas]
We should have a sitecore jss blazor sdk. Today there are:
React
Angular 
Vue
Nextjs 

And... Where is Blazor 🤔

**Comment by GuitarRich** [comment]
Wouldn't that be its own thing? Like the netcore rendering host? JSS === JavaScript Services....

**Reply by GoranHalvarsson** [reply]
No not really. It's blazor on the client side - blazor wasm

**Comment by art-alexeyenko** [comment]
After a while, some good news (alas not from JSS side) - we have ASP.NET Core SDK now for all the .net needs:
https://doc.sitecore.com/xmc/en/developers/asp-net/latest/asp-net-core-sdk/installation.html

Not quite Blazor, and JSS will continue focusing on nodeJS frameworks, but our .net professionals in the community should now have more ways to XMCloud.
